content/en/docs/Writing policies/match-exclude.md

```diff
@@ -238,6 +238,8 @@ spec:
     - name: match-criticals-except-given-rbac
       match:
         resources:
+	  kinds:
+	  - Pod
```

Comment:
What are these red colored blocks before the text?

Reply:
Exactly, I also don't know. But, I think it's representing `tab` space.

Reply:
Remove that please. I think just deleting it and adding it normally should fix it.

Reply:
In the YAML file, indentation is necessary. And if I remove this indentation i.e. tab space, an error will occur in the YAML file.

Reply:
You need to check your IDE. A tab should be rendered as two spaces in YAML. This suggests it's rendered as a tab character which it shouldn't be.

Reply:
Btw, I talked about that tab space from a general point of view. But, actually, I have used 2 spaces. 

![Screenshot (587)](https://user-images.githubusercontent.com/65091408/123192128-ef167300-d4bf-11eb-83b9-9118849494de.png)

Reply:
That screenshot doesn't distinguish between those characters. VS Code, for example, will with the YAML addon.

Reply:
Yes, @chipzoller when I open it in VS Code, I can easily visualize the exact problem. I don't why IDE to IDE these indentation differs. 

![Screenshot (589)](https://user-images.githubusercontent.com/65091408/123194632-77971280-d4c4-11eb-94fa-76cf267e98e5.png)